errors.go

```diff
@@ -114,26 +112,6 @@ func (se stackErr) Error() string {
 	return se.err.Error()
 }
 
-// Format controls the optional display of the stack trace. Use %+v to output the stack trace, use %v or %s to output
-// the wrapped error only, use %q to get a single-quoted character literal safely escaped with Go syntax for the wrapped
-// error.
-func (se stackErr) Format(s fmt.State, verb rune) {
```

Comment:
Question: Any reason why we wouldn't deprecate this before removing?

Reply:
Also, I didn't know this existed. Nothing internally used it at all? Was this complying with some standard interface? Very weird.

Reply:
https://pkg.go.dev/fmt#Formatter

We're not concerned about people calling it directly—it's probably a safe assumption that no one has ever done that before. We're concerned about calls to `fmt` functions with the `%+v` verb.